setup.py

```diff
@@ -7,12 +7,13 @@
 setuptools.setup(
     name="pello",
```

Comment:
For our purposes, we'd like to have the P uppercase here, to demonstrate how to package packages with such names.